jobs/src/main/scala/dcos/metronome/jobrun/JobRunModule.scala

```diff
@@ -1,9 +1,9 @@
 package dcos.metronome.jobrun
 
-import akka.actor.{ ActorContext, ActorSystem, Props }
+import akka.actor.{ActorContext, ActorSystem, Props}
```

Comment:
apparently our formatter wants spaces here ... running compile reformats a bunch of code.